library/L2_HAL/sensors/environment/light/temt6000x01.hpp

```diff
@@ -0,0 +1,53 @@
+#pragma once
+
+#include "L2_HAL/sensors/environment/light.hpp"
+
+#include "L1_Peripheral/adc.hpp"
+#include "utility/log.hpp"
+
+namespace sjsu
+{
+class Temt6000x01 final : public Light
+{
+ public:
+  /// Pull-down resistor
+  static constexpr uint32_t kResistance = 10'000;
```

Comment:
This is not a constant in the datasheet, meaning other developers may use a different resistor. We shouldn't assume everyone is using 10k. This should be a constructor parameter.